webrtc/README.md

```diff
@@ -0,0 +1,12 @@
+# WebRTC
+
+This directory contains the WebRTC test suite.
+
+## Attribution
+
+Some data channel tests are based on the [data channel conformance test
+suite][nplab-webrtc-dc-playground] of the Network Programming Lab of the Münster
+University of Applied Sciences. We would like to thank Peter Titz, Felix Weinrank and Timo
+Völker for allowing us to make their test cases available under the W3C license.
```

Comment:
Note that per https://github.com/web-platform-tests/wpt/issues/11009 the license may change, as allowed for by the terms in CONTRIBUTING.md.

To not have to update this text later, may I suggest not mentioning the license but instead sometime like "agreeing to contribute their test cases to this repository"?